### DISCUSSION TITLE
Legacy Form and Other donation form is not working for guest users

### DISCUSSION THREAD

**Discussion by dightinfotechcom** [impress-org/givewp#7230] [Developer Q&A]
Hi there!

Hope everything is fine. 

I was just checking compatibility with browser and I found something

here is link of video which you can check this out. 

https://www.loom.com/share/e1166b8531064672a19f623d650a4fce

While checking on safari brower browser legacy form and other donation form is not working for guest users. 
it gives some error, Sometimes of Customer id error sometimes it is giving of amount not valid. 

And while these error appear, total donation amount changes to default donation amount and selected amount is different. 
I tried to figure out but couldn't find the reason for this. Can you check this please.

here is a form of my donation form and legacy form

https://dev.dightinfotech.com/easymerchant/donations/legacy-form/ === legacy form
https://dev.dightinfotech.com/easymerchant/donations/20/ ==== Donation form

check this in safari browser as a guest user, while I am logged in, everything working fine. 

Thanks.

**Comment by kjohnson** [comment]
Hey @dightinfotechcom. I wasn't able to replicate this locally. If you do a clean install of GiveWP are you still getting this error?

**Reply by dightinfotechcom** [reply]
Yes @kjohnson. I am still facing. I installed fresh new plugin 
![image](https://github.com/impress-org/givewp/assets/148416099/34e4a68d-c499-4a32-8786-8a328a0fcbe4)
can you check this please?
https://dev.dightinfotech.com/easymerchant/donations/legacy-form/ === legacy form
https://dev.dightinfotech.com/easymerchant/donations/20/ ==== Donation form

I have this link you can try it. Open these link on **safari browser** please.

**Reply by dightinfotechcom** [reply]
Hi @kjohnson 
I resolve this issues. 

Can you tell me what function will be used for create renewal through webhook. I want to apply this function so renewal should automatically created through webhook. and renewal entries should visible in table. 

Please reply.
